odinmain/src/main/kotlin/me/odinmain/features/impl/nether/KuudraReminders.kt

```diff
@@ -42,7 +45,8 @@ object KuudraReminders : Module(
             PlayerUtils.alert("Fresh Tools", displayText = displayText, playSound = playSound)
         }
 
-        onMessage(Regex("Used Extreme Focus! \\((\\d+) Mana\\)"), { manaDrain && enabled }) {
+        onMessage(Regex("Used Extreme Focus! \\((\\d+) Mana\\)"), { manaDrain && enabled}) {
+            if (onlyKuudra && !KuudraUtils.inKuudra) return@onMessage
```

Comment:
Move the check to inside the lambda in the onMessage
Resolve conflicts
and ill merge